docs/website/docs/general-usage/incremental-loading.md

```diff
@@ -33,19 +28,15 @@ using `primary_key`. Use `write_disposition='merge'`.
 
 </div>
 
-The "write disposition" you choose depends on the data set and how you can extract it.
+The "write disposition" you choose depends on the dataset and how you can extract it.
```

Comment:
This is a strange change as I think both terms are valid

Reply:
We use dataset throughout the docs and in the code, so I would say it is good, no?